csharp-basics/exercises/Collections/UniquValues/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace UniqueValues
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var values = new List<string> { "Hi", "Meow", "Hello", "Meow", "Hi!", "Meow", "Hi", "Bye" };
+
+            var uniqueString = values.Distinct().ToArray();
```

Comment:
Nesanāk kā piemērā  //ToDo: Given a non-empty list of strings, return a list that contains only unique (non-duplicate) strings.
            //ToDo: Example: ["abc", "xyz", "klm", "xyz", "abc", "abc", "rst"] → ["klm", "rst"]